WordPress/UITestsFoundation/Screens/Login/Unified/PrologueScreen.swift

```diff
@@ -18,7 +18,7 @@ public class PrologueScreen: ScreenObject {
         try super.init(
             expectedElementGetters: [continueButtonGetter, siteAddressButtonGetter],
             app: app,
-            waitTimeout: 3
+            waitTimeout: 7
```

Comment:
increased this to 7 (what most other screens are using) because one test failed in this spot ([link](https://buildkite.com/organizations/automattic/analytics/suites/jetpack-ios-ui-tests-ipad/runs/12e70629-5a18-414c-85a6-f9a25c9fa452#failed)) after `dismissNotificationAlertIfNeeded` was removed. 

i suspect that with that removed, the 3-second timeout is no longer enough here (previously it was 3 seconds + the actions in `dismissNotificationAlertIfNeeded` that wasn't really necessary anymore)